Project 5 For John Beverly

```diff
@@ -59,345 +59,388 @@ Group 2 contains Emanuel, Katie, Josh V (commonly know as V by group member), Ca
 Please note we DID check this against SHACL playground :) 
 enjoy, if possible.
 
-
-1st constraints begins below
-
-@prefix ex: <http://example.org/> .
-
+1.	Constraint for Asymmetry 
 @prefix sh: <http://www.w3.org/ns/shacl#> .
 
-@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix ex: <http://example.com/> .
 
 @prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
 
-@prefix schema: <http://schema.org/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
-ex:RegistrationShape
+ex:AsymmetryShape
 
 a sh:NodeShape ;
 
-sh:targetClass ex:Registration ;
+sh:targetClass ex:relation ;
 
 sh:property [    
 
-sh:path ex:lineRegistration ;    
+sh:path rdf:value ;    
 
-sh:order ex:lineRegistrationOrder ;    
+sh:minCount 1 ;    
 
-sh:node ex:RegistrationShape ; 
+sh:maxCount 1 ;    
+
```

Comment:
so we are saying that ex:relation will have exactly one rdf:value, right? In which sense is this asymmetric? In the sense that in that path we will not get any other value? But I am not sure that this works, because then it also stops us from having any other value at all. I.e. say that we have the path "hugging". This is stopping me from hugging or being hugged from more than one person, while what you want to say is that no one can hug me back (sad), but you don't want to prevent me from hugging more than one person.